apps/auth-admin-web/infra/auth-admin-web.ts

```diff
@@ -0,0 +1,74 @@
+import { service, ServiceBuilder } from '../../../infra/src/dsl/dsl'
+
+export const serviceSetup = (): ServiceBuilder<'auth-admin-web'> => {
+  return service('auth-admin-web')
+    .namespace('identity-server-admin')
+    .image('auth-admin-web')
+    .env({
+      IDENTITYSERVER_SCOPE:
+        'openid profile auth-admin-api.full_control offline_access',
+      IDENTITYSERVER_CLIENT_ID: 'ids-admin',
```

Comment:
CLIENT_ID was removed in  [5f4360b](https://github.com/island-is/island.is/pull/7350/commits/5f4360bbf4cc649aa87167141a3d743e60d4ac0c) because it was mentioned as unused